query_postal_codes_heatmaps_api.py

```diff
@@ -0,0 +1,103 @@
+import random
+from collections import defaultdict
+import statistics
+
+from datetime import datetime, timedelta, date
+import matplotlib.pyplot as plt
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+
+
+def build_api_query(date: date, tile_ids: list) -> str:
+    api_request = (
+        BASE_URL
+        + "/heatmaps/dwell-density/daily/{0}".format(date.isoformat())
+        + "?tiles="
```

Comment:
It would be safer to let `requests` handle the parameters for you in case you have special characters that need to be URL-encoded, etc.: https://requests.readthedocs.io/en/master/user/quickstart/#passing-parameters-in-urls

Reply:
Much better indeed. Thanks for the tip 
20acc57d0ae04f1f1a62782317de1a9621d938e3